api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -54,19 +48,19 @@
 
   private final Provider<GoogleCredential> googleCredentialProvider;
   private final Provider<WorkbenchConfig> configProvider;
-  private final Provider<MailService> mailServiceProvider;
   private final HttpTransport httpTransport;
   private final GoogleRetryHandler retryHandler;
+  private final Provider<MailService> mailServiceProvider;
```

Comment:
This can be reverted along with the deletion of line 57 to keep the # of changes smaller.